### PR TITLE
Ccp 1059

### PR DESCRIPTION
**Pull request by Carsten89**
# Pull Request Template

## Description
Added
- `withThemeComponents` connector
- `withProductContext` connector
- `ProductCharacteristics` component

## Type of change

- [ ] Bug fix
- [x] New feature
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,13 @@ This package follows semver.
   - `/connectors/withPageProductId` - Provides decoded `productId` read from a pathname of a product page. [read documentation](https://github.com/shopgate-professional-services/pwa-extension-kit/blob/master/src/connectors/README.md#withPageProductId)
   - `/connectors/withPageState` - Provides `isVisible` and `isLoading` booleans. Values are taken from a page state. [read documentation](https://github.com/shopgate-professional-services/pwa-extension-kit/blob/master/src/connectors/README.md#withPageState)
   - `/connectors/withUser` - Provides `user` object with user data [read documentation](https://github.com/shopgate-professional-services/pwa-extension-kit/blob/master/src/connectors/README.md#withUser)
```
Comment (sznowicki) [inline README.md:23]: withThemeComponents + withProductContext mention is missing here

**Review by sznowicki — COMMENTED** [review]
package-lock should stay

File: src/connectors/README.md [hunk 2]
```diff
@@ -156,3 +156,61 @@ MyComponent.propTypes = {
...
+        finishTimeout={200}
+      />
+  );
+}
+
+export default withProductContext(MyComponent);
+```
+
+### withThemeComponents
+Connects provided component with some components provided by the Theme.
```
Comment (sznowicki) [inline src/connectors/README.md]: Some background info would be helpful here. "with components provided by the Theme. Theme components are theme specific - maintain functionality with theme specific UI"

File: src/connectors/README.md [hunk 3]
```diff
@@ -156,3 +156,61 @@ MyComponent.propTypes = {
...
+
+#### Props provided
+- `AppBar (ReactComponent)` - <AppBar> Component from the Theme
```
Comment (sznowicki) [inline src/connectors/README.md]: Please add little explanation what are those and why/how would I use them.

File: src/components/index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import ProductCharacteristics from './ProductCharacteristics';
```
Comment (sznowicki) [inline src/components/index.js:1]: This file is not covered. Please provide unit test.

**Review by sznowicki — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Carsten89) [reply README.md:23]: done

Comment (Carsten89) [reply README.md:23]: done

File: src/connectors/README.md (hunk 2 above)
Comment (Carsten89) [reply src/connectors/README.md]: done

File: src/components/index.js (hunk 4 above)
Comment (Carsten89) [reply src/components/index.js:1]: done

File: src/connectors/README.md (hunk 3 above)
Comment (Carsten89) [reply src/connectors/README.md]: done

**Review by sznowicki — APPROVED** [review]

**Pull request merged**